test/util/load_sku_packs.py

```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
                     subprocess.call("cd on-skupack; ./build-package.bash "
                                     + skus + " " + skus + " >/dev/null 2>&1", shell=True)
             break
-        # upload SKU packs to ORA
-        print "**** Loading SKU Packs to server"
+
+    def test02_upload_sku_packs(self):
+        # upload SKU packs to RackHD
         for subdir, dirs, files in os.walk('on-skupack/tarballs'):
             for skupacks in files:
                 print "\n**** Loading SKU Pack for " + skupacks
-                fit_common.rackhdapi("/api/1.1/skus/pack", action="binary-post",
+                fit_common.rackhdapi("/api/2.0/skus/pack", action="binary-post",
                                      payload=file(fit_common.TEST_PATH + "on-skupack/tarballs/" + skupacks).read())
             break
         print "\n"
+
+    def test03_verify_sku_packs(self):
         # check SKU directory against source files
-        errorcount = 0
+        errorcount = ""
```

Comment:
Please rename the `errorcount` to something like `error_message` as it is not a number